### PR TITLE
docs: update README instructions

### PR DESCRIPTION
**Pull request by tcrasset**
- docs: update docs with new installation instructions
- chore: adding CODEOWNERS file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# How to contribute ?
+
+This is a guide intended for developers wanting to lend a hand.
+It contains detailed instructions on how to setup your development environment and
+the prerequisites needed to work on this project.
+## Prerequisites
```
Comment (charlax) [inline CONTRIBUTING.md]: Can you run prettier on this file?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+If you have never created an R package before, it is good idea to follow the tutorial here: https://r-pkgs.org/whole-game.html
+
+## Installing deps
+
+```bash
+make install
+```
+
+Make sure things are properly installed>
```
Comment (charlax) [inline CONTRIBUTING.md]:
```suggestion
Make sure things are properly installed:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# How to contribute ?
+
+This is a guide intended for developers wanting to lend a hand.
+It contains detailed instructions on how to setup your development environment and
+the prerequisites needed to work on this project.
+## Prerequisites
+
+Make sure you have R installed. On macOs:
```
Comment (charlax) [inline CONTRIBUTING.md]:
```suggestion
Make sure you have R installed. On macOS:
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+Your system is ready to build packages!
+```
+
+## Quickstart
+
+Install with:
+
+```R
+# type="source" prevents a warning about R version
+install.packages("...", repos=NULL, type="source")
```
Comment (charlax) [inline CONTRIBUTING.md:42]: Replace with Github release URL?

**Review by charlax — APPROVED** [review]
Nice!

File: CONTRIBUTING.md (hunk 4 above)
Comment (tcrasset) [reply CONTRIBUTING.md:42]: Well that's for installation from source, if we are developing, we don't necessarily want to take the released version ?

**Pull request merged**